Neos.Utility.ObjectHandling/Classes/TypeHandling.php

```diff
@@ -51,9 +51,9 @@ public static function parseType(string $type): array
             throw new InvalidTypeException('Found an invalid element type declaration. A type "' . var_export($type, true) . '" does not exist.', 1264093630);
         }
 
-        $typeWithoutNull = self::stripNullableType($matches['type']);
-        $isNullable = $typeWithoutNull !== $matches['type'];
-        $type = self::normalizeType($typeWithoutNull);
+        $typeWithoutNull = self::stripNullableType($type);
+        $isNullable = $typeWithoutNull !== $type || $type === 'null';
```

Comment:
The `$type === 'null'` is a borderline edge-case tbh - no one should annotate something with just the type `null` and in PHP types that is even invalid. Anyway, if you do have `@param null $foo` and check if it is "nullable" I guess the expectation is it to be true?

Reply:
I guess so…